CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unpublished
+*larva-tokens -Add compat tokens for variety
```

Comment:
Need a space after the `*` for markdown formatting

Reply:
> Check the regression test results by downloading the `backstop-results` artifact at the bottom of this page: https://github.com/penske-media-corp/pmc-larva/actions/runs/1784847025
> 
> Open the html_report/index.html file in a browser to see which tests are failing. This is a new system, so it's possible there is a false negative.

It looks like all of the tests are failing. When I open the file. Some of the text appears to shift based on the diff. What could I have added that changed that?

Reply:
Thanks! This has been fixed.